UnitsNet/GeneratedCode/UnitAbbreviationsCache.g.cs

```diff
@@ -127,28 +130,40 @@ public partial class UnitAbbreviationsCache
                 ("en-US", typeof(BitRateUnit), (int)BitRateUnit.BitPerSecond, new string[]{"bit/s", "bps"}),
                 ("en-US", typeof(BitRateUnit), (int)BitRateUnit.BytePerSecond, new string[]{"B/s"}),
                 ("en-US", typeof(BitRateUnit), (int)BitRateUnit.ExabitPerSecond, new string[]{"Ebit/s"}),
+                ("en-US", typeof(BitRateUnit), (int)BitRateUnit.ExabitPerSecond, new string[]{"Ebps"}),
```

Comment:
Well this is awesome to see, how old unit definitions now get populated new abbreviations !
And as you say, extra row vs array makes no practical difference here.

Reply:
Yeah, let's just hope people don't get surprised in a bad way as this opens the possibility for many new AmbiguousUnitParseExceptions..

Reply:
Good point, yeah that would actually be a nice list to have.